### PR TITLE
Upgrade gatekeeper version

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: modules/kubernetes/opa-gatekeeper/templates/gatekeeper-values.yaml.tpl [hunk 1]
```diff
@@ -1,7 +1,6 @@
 postInstall:
   labelNamespace:
     enabled: false
-experimentalEnableMutation: true
```
Comment (NissesSenap) [inline modules/kubernetes/opa-gatekeeper/templates/gatekeeper-values.yaml.tpl:4]: is this the new default value?

**Review by NissesSenap — APPROVED** [review]

File: modules/kubernetes/opa-gatekeeper/templates/gatekeeper-values.yaml.tpl (hunk 1 above)
Comment (phillebaba) [reply modules/kubernetes/opa-gatekeeper/templates/gatekeeper-values.yaml.tpl:4]: Yes it has been removed for another value, mutations are now enabled by default.

**Review by NissesSenap — APPROVED** [review]
Isnt this a breaking change?

**Comment by phillebaba** [comment]
No CRDs are managed by a job in the chart. Upgrade should be seamless.

**Pull request merged**